src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -559,3 +565,79 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory wi
     }.transform(tree)
   }
 }
+
+class OrganizeImportsSpacer(val oiInstance: OrganizeImports) {
+  import oiInstance.global._
+  import scala.tools.nsc.interactive.Global
+
+  private def topMostClassOrModule(stats: List[Tree]): Option[ImplDef] = stats.find {
+    case classDefOrModuleDef: ImplDef => true
+    case _ => false
+  }.asInstanceOf[Option[ImplDef]]
+
+  private def linesDiffInOrigFile(classOrModule: Option[ImplDef], importt: Option[Import]): Option[Int] =
+    for {
+      cm <- classOrModule
+      imp <- importt
+      sourceFile = cm.pos.source
+      cmLine = sourceFile.offsetToLine(cm.pos.start)
+      impLine = sourceFile.offsetToLine(imp.pos.end)
+    } yield cmLine - impLine
+
+  private def lastImportInPackageScope(stats: List[Tree]): Option[Import] = topMostClassOrModule(stats).flatMap { topMost =>
+    val index = stats.indexOf(topMost)
+    stats.take(index).reverse.dropWhile { !_.isInstanceOf[Import] }.headOption.asInstanceOf[Option[Import]]
```

Comment:
The part after reverse can be replaced by `collectFirst { case imp: Import => imp }`, couldn't it?